processor/src/test/java/org/mapstruct/ap/test/nestedbeans/NestedSimpleBeansMappingTest.java

```diff
@@ -41,6 +44,13 @@
 @RunWith(AnnotationProcessorTestRunner.class)
 public class NestedSimpleBeansMappingTest {
 
+    @Rule
+    public final GeneratedSource generatedSource = new GeneratedSource().addComparisonToFixtureFor(
```

Comment:
What do you mean? That I don't call the `getter` and check if they are actually what we expect? This tests for this actually work with the overridden `equals` on the Dtos. I can change the assertions to actually call the `getters` instead